src/etc/inc/services.inc

```diff
@@ -2214,7 +2215,7 @@ function dyndnsCheckIP($int) {
 		$ip_result_page = curl_exec($ip_ch);
 		curl_close($ip_ch);
 		$ip_result_decoded = urldecode($ip_result_page);
```

Comment:
Also, it's worth noting that this `urldecode` call is both unnecessary and inappropriate, as no such service would return URL-encoded content as part of a response body. One should simply pass `$ip_result_page` into `$preg_match` directly, and not "decode" the IP response with a codec which would never be used in that context.

(This sounds pedantic, but would theoretically cause problems in the event that a user is doing DDNS on their device's IPv6 link-local address.)